internal/audit/file_test.go

```
@@ -397,3 +410,182 @@ func TestEngineConcurrentRecords(t *testing.T) {
+	key := bytes.Repeat([]byte{0xAA}, 32)
+	ce, err := crypto.NewEngine(key, log.NewNoOpLogger())
+	if err != nil {
+		t.Fatal(err)
+	}
+	fp := crypto.FingerprintBytes(key)
+
+	f, err := newFile(dir, ce, log.NewNoOpLogger(), KeyModeSimple, fp)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)